lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,5 @@
-
 def palindrome_permutation?(string)
-  
+  h = Hash.new { |h, k| h[k] = 0 }
```

Comment:
A good improvement here would be to use a variable name that is more descriptive of what the hash is being used for and/or is meant to store. Generally avoid using single-letter variable names except for extremely common idioms e.g. using `i` and `j` for loop control variables.